cmake/Helpers.cmake

```diff
@@ -1,13 +1,16 @@
-#####################
-## Helpers.cmake
-## Top contributors (to current version):
-##   Mathias Preiner, Aina Niemetz, Andres Noetzli
-## This file is part of the CVC4 project.
-## Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
-## in the top-level source directory and their institutional affiliations.
-## All rights reserved.  See the file COPYING in the top-level source
-## directory for licensing information.
+###############################################################################
+# Top contributors (to current version):
+#   Mathias Preiner, Aina Niemetz, Andres Noetzli
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
 ##
```

Comment:
Should we have something in here?

Reply:
Yes, we should. But adding missing file descriptions is out of scope for this PR. I will need everybody's help for this.